npm/design-system/src/components/FileExplorer/FileExplorer.spec.tsx

```diff
@@ -0,0 +1,25 @@
+import { mount } from '@cypress/react'
+import React from 'react'
+import { FileExplorer } from './FileExplorer'
+import { FileLike } from './types'
+
+const files: FileLike[] = [
+  {
+    name: 'foo/bar/foo.spec.js',
+    onClick: (e, foo) => {
+    },
+    isOpen: false,
+  },
+  // @ts-ignore
+  { name: 'bar/foo.spec.tsx' },
```

Comment:
why do we need ts-ignore?

Reply:
This is wrong. It's because I was hacking on this file and hadn't finished the types.